[exporter/kafka]Decide the Kafka topic based on the value of the attribute. (#31809)
open-telemetry/opentelemetry-collector-contrib: exporter/kafkaexporter/README.md

- `resolve_canonical_bootstrap_servers_only` (default = false): Whether to resolve then reverse-lookup broker IPs during startup.
- `client_id` (default = "sarama"): The client ID to configure the Sarama Kafka client with. The client ID will be used for all produce requests.
- `topic` (default = otlp_spans for traces, otlp_metrics for metrics, otlp_logs for logs): The name of the kafka topic to export to.

> - `topic_from_attribute`: Specify the attribute name when you want to dynamically obtain the topic name from an attribute. Within the same transmission unit, if there are multiple values for the same attribute, use the first value that can be obtained.

Reviewer: ```suggestion
- `topic_from_attribute` (default = ""): Specify the resource attribute whose value should be used as the message's topic. This option, when set, will take precedence over the default topic.
```
Wording nit

Author: thanks!

> - `topic_from_attribute` (default = ""): Specify the resource attribute whose value should be used as the message's topic. This option, when set, will take precedence over the default topic. If topic_from_attribute is not found in the attributes, the value set in topic will be used instead. 

Reviewer: ```suggestion
- `topic_from_attribute` (default = ""): Specify the resource attribute whose value should be used as the message's topic. This option, when set, will take precedence over the default topic. If `topic_from_attribute` is not set, the message's topic will be set to the value of the configuration option `topic` instead. 
```
Minor wording nit.
